jarbas/api/views.py

```diff
@@ -26,9 +26,16 @@ def get(self, request):
         values = map(self.request.query_params.get, params)
         filters = {k: v for k, v in zip(params, values) if v}
 
-        # filter queryset
-        if self.request.query_params.get('suspicions'):
+        # filter suspicions
+        if self._bool_param('suspicions'):
             self.queryset = self.queryset.suspicions()
+
+        # filter reimbursement in latest dataset
+        in_latest = self._bool_param('in_latest_dataset')
+        if in_latest is not None:
```

Comment:
Quick question: Why do we use `None` instead of a condition containing `True` or `False`? Maybe I got lost on the logic here =)